src/components/views/voip/PipView.tsx

```diff
@@ -54,6 +54,16 @@ const SHOW_CALL_IN_STATES = [
     CallState.WaitLocalMedia,
 ];
 
+/**
+ * Callback to create the pip content view.
+ * Passes down the resize and move callbacks to the child component.
+ * {@link ./PictureInPictureDragger.tsx contains an example usage of it}
```

Comment:
is that the *only* place that uses this interface? If so, shouldn't it be defined in that file?

In fact, it looks like there is already an `IChildrenOptions` there which this should reference rather than redefining the structure of the object.